server/routes/team.router.js

```diff
@@ -0,0 +1,105 @@
+const express = require('express');
+const pool = require('../modules/pool');
+const router = express.Router();
+const {
+  rejectUnauthenticated,
+} = require("../modules/authentication-middleware");
+
+router.get('/all', (req, res) => {
+  // console.log('in teams GET router');
+  let queryText = `
+    SELECT "users".name as userName, 
+      "usersTeams"."userId", "usersTeams"."teamId", 
+      "teams".name as "teamName", "teams".id as "teamId", "teams"."captainId", "teams"."accessCode"
+    FROM "users"
+    JOIN "usersTeams" ON "users".id = "usersTeams"."userId"
+    JOIN "teams" ON "teams".id = "usersTeams"."teamId";`
+  pool
+    .query(queryText)
+    .then((result) => {
+      res.send(result.rows);
+    })
+    .catch(err => {
+      console.log('Error in Team GET', err);
+      res.sendStatus(500)
+    })
+});
+
+router.get('/access', (req, res) => {
```

Comment:
**CHANGE REQUIRED**

This is not secure. Basically your sending users a list of all available access code, so anyone has access to join anyone's team. 

We should talk through this, but in short: a user should `POST` their access code, and the server should check if it's a match (just like how we do for passwords). If this was a more security-sensitive app, I'd have you encrypt the access code too, but we're probably ok without that here.